### PR TITLE
Add pycocotools to Requirements, Installation sections

### PR DESCRIPTION
**Pull request by philferriere**
Let's try this again from the right branch this time...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,13 +159,21 @@ You can also [join our team](https://matterport.com/careers/) and help us build
...
 If you use Docker, the model has been verified to work on
 [this Docker container](https://hub.docker.com/r/waleedka/modern-deep-learning/).
 
+The package `pycocotools` requires `cython` and a C compiler to install correctly. See below for further instructions.
+
 ## Installation
 1. Clone this repository
 2. Download pre-trained COCO weights from the releases section of this repository.
+3. Install `pycocotools` as follows:
+    - On Linux, run `pip install git+https://github.com/waleedka/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI`
```
Comment (RyanGordon) [inline README.md]: I think this should be `pip install git+https://github.com/waleedka/coco.git#egg=pycocotools&subdirectory=PythonAPI`

**Comment by philferriere** [comment]
I'm pretty sure @RyanGordon is correct.

**Comment by waleedka** [comment]
The pip commands didn't work for me. I changed it to simply link to the pycocotools repos so people can read the setup instructions there. Also, added language to clarify that pycocotools is only needed if you plan to train on MS COCO.

**Comment by waleedka** [comment]
I figured out the issue with the pip command. I had to remove the `egg` attribute and then it worked. So the command becomes:
```
pip3 install git+https://github.com/waleedka/coco.git#subdirectory=PythonAPI
```

I don't have Windows to verify the other command. @philferriere can you pleavse verify that this works? Is the ^ in the egg name required? And in fact, is the egg param required? On the Linux version it isn't and actually causes the command to fail.
```
pip3 install git+https://github.com/philferriere/cocoapi.git#egg=pycocotools^&subdirectory=PythonAPI
```

**Comment by philferriere** [comment]
Most of us get our deep learning python setup through Anaconda. Anaconda uses `pip`, not `pip3`, nor does Anaconda register an alias for pip called `pip3`, at least not on Windows. Therefore, I would not recommend using pip3. It will fail for sure on Windows with a "regular" anaconda setup:

```
(dlwin36test1) pferr@MSI d:\repos
$ pip3 install git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI
'pip3' is not recognized as an internal or external command,
operable program or batch file.
```

The ^ is here to escape the & character on the Windows command line. As it turns out, because pycocotools's `setup.py` does the right thing, the `egg=pycocotools` param is unnecessary. The commands `pip install git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI` and `pip install git+https://github.com/philferriere/cocoapi.git#egg=pycocotools^&subdirectory=PythonAPI`do the exact same thing on Windows:

```
(dlwin36test1) pferr@MSI d:\repos
$ pip install git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI
Collecting git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI
  Cloning https://github.com/philferriere/cocoapi.git to c:\users\pferr\appdata\local\temp\pip-ydfep31c-build
Installing collected packages: pycocotools
  Running setup.py install for pycocotools ... done
Successfully installed pycocotools-2.0

(dlwin36test1) pferr@MSI d:\repos
$ python
Python 3.6.3 |Anaconda, Inc.| (default, Oct 27 2017, 12:22:41) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pycocotools
>>> pycocotools.__file__
'D:\\toolkits.win\\anaconda3-5.0.1\\envs\\dlwin36test1\\lib\\site-packages\\pycocotools\\__init__.py'
>>>
```

The major difference between your fork and mine for pycocotools is in the Makefile for cython and the fix for Python 3. Feel free to `diff` the files to see what the differences are.

**Pull request merged**